### ISSUE TITLE
Has trouble compiling local dependencies

### ISSUE THREAD

**Issue by slightknack** [rust-build/rust-build.action#2]
Currently, I'm compiling a binary with two components: a core library, and a CLI that wraps it. The core library is a rust crate specified as a path dependency:

```
./project
├── passerine (core library)
│   ├── Cargo.toml
│   └── src/ ...
└── aspen (cli tool)
    ├── Cargo.toml
    └── src/ ...
```

When building a release for the Passerine repository, I would like to compile the `aspen` CLI crate rather than the `passerine` core library crate. I currently have a GH action that uses `rust-build.action`:

```yml
name: Release

on:
  release:
    types: [created]

jobs:
  release:
    name: Release ${{ matrix.target }}
    runs-on: ubuntu-latest
    strategy:
      fail-fast: false
      matrix:
        target: [x86_64-pc-windows-gnu, x86_64-unknown-linux-musl, x86_64-apple-darwin]
    steps:
      - name: Checkout Passerine
        uses: actions/checkout@v2
        with:
          path: passerine

      - name: Checkout Aspen
        uses: actions/checkout@v2
        with:
          repository: vrtbl/aspen
          path: aspen

      - run: cd aspen

      - name: Compile and Release
        uses: Douile/rust-build.action@v0.1.26
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          RUSTTARGET: ${{ matrix.target }}
```

Looking at the shell script, it doesn't seem like it matters that I set up directories side-by-side, and for this reason the build fails. How should I use `rust-build.action` to compile and build a release for `aspen` when running an action in Passerine's repo?

Thanks!

**Comment by Douile** [comment]
There is currently no way to change the source dir, it just tries to build with cargo at the root of the repository.
I will add add something like `SRC_DIR` to allow you to build a relative path from repo root.

**Comment by slightknack** [comment]
Thanks for looking into this!

**Comment by Douile** [comment]
Should be fixed in v1.0.0

**Comment by slightknack** [comment]
I'll try to update to v1.0.0. If anything else goes wrong, I'll let you know - thanks again!
